zerver/management/commands/realm_linkifiers.py

```diff
@@ -41,24 +41,24 @@ def handle(self, *args: Any, **options: str) -> None:
         realm = self.get_realm(options)
         assert realm is not None  # Should be ensured by parser
         if options["op"] == "show":
-            print(f"{realm.string_id}: {all_realm_filters().get(realm.id, [])}")
+            print(f"{realm.string_id}: {linkifiers_from_all_realms().get(realm.id, [])}")
             sys.exit(0)
 
         pattern = options["pattern"]
         if not pattern:
-            self.print_help("./manage.py", "realm_filters")
+            self.print_help("./manage.py", "realm_linkifier")
```

Comment:
I think there's probably a prep commit you could do to make this access its own command name via sys.args[1] or something rather than hardcoding?  Not sure how easy this is with Django's management command system, but that'd be a good sweep to do.

Reply:
I added a prep commit which does that sweep, and yet another one before it as a small correction 😄